ray-curator/ray_curator/models/configs/internvideo2_mm_config_bert.json

```diff
@@ -0,0 +1,25 @@
+{
```

Comment:
I'm thinking if these configs should also live inside the `git+https://github.com/suiyoubi/InternVideo.git@curator#subdirectory=InternVideo2/multi_modality",` ?